graphpype/tests/test_utils_img.py

```diff
@@ -0,0 +1,43 @@
+from graphpype.utils_img import (return_data_img_from_roi_mask)
+
+import os
+import numpy as np
+import nibabel as nib
+
+try:
+    import neuropycon_data as nd
```

Comment:
what is this?

Reply:
What do you mean? The inclusion of neuropycon_data? We had this discussion on slack about including data in another package + you told me to remove the assertion from the test and add a try catch at the beginning of the test files. This is already the case in test_utils_cor.py:
https://github.com/neuropycon/graphpype/blob/ab13e9622956ac6e88deb1fc13515f3253606701/graphpype/tests/test_utils_cor.py#L15-L19

and test_utils_net.py

https://github.com/neuropycon/graphpype/blob/ab13e9622956ac6e88deb1fc13515f3253606701/graphpype/tests/test_utils_net.py#L9-L13

Reply:
> What do you mean? The inclusion of neuropycon_data? We had this discussion on slack about including data in another package 

okay, this is not what I understood from the discussion on slack. From what I got, you will host the data on another github repository but this will not add another dependency. What you need to do is to fetch the data automatically on demand if the data directory does not exist.